package.json

```diff
@@ -75,6 +75,8 @@
   "dependencies": {
     "@elastic/elasticsearch": "7.9.0",
     "@elastic/elasticsearch-mock": "0.3.0",
+    "@next/mdx": "10.0.6",
+    "@octokit/webhooks": "8.0.2",
```

Comment:
I don't think these are used in this PR.

Reply:
It's not, but I had to include them, otherwise it'll fail the tests for missing packages 🤷

Reply:
how can you fail tests if those packages are not used in master?

Reply:
oh, I see that `@next/mdx` is already included in master, and here it seems you're updating it to version `10.0.6`.

Reply:
Also, I ran our tests without `@octokit/webhooks` and it didn't fail.
Should we update `@next/mdx` in  a different PR maybe?

cc: @humphd

Reply:
Yes, cc @chrispinkney, who is managing this.  Let's not do it here, please revert this and we'll do it in a new PR.